src/battle-scene.ts

```diff
@@ -2965,7 +2965,7 @@ export default class BattleScene extends SceneBase {
       biome: this.currentBattle ? getBiomeName(this.arena.biomeType) : "",
       wave: this.currentBattle?.waveIndex || 0,
       party: this.party ? this.party.map(p => {
-        return { name: p.name, level: p.level };
+        return { name: p.name, type: p.getTypes(), level: p.level, currentHP: p.hp, maxHP: p.getMaxHp(), status: p.status };
```

Comment:
```suggestion
        return { name: p.name, types: p.getTypes(), level: p.level, currentHP: p.hp, maxHP: p.getMaxHp(), status: p.status };
```

FYI, getTypes() will not return the Pokemon's tera type if applicable.
If it's something you would potentially be interested in getting you should check the optional parameters of the function, or alternatively getTeraType()

Reply:
Thank you for the tip. I added getTeraType() and changed "type" to "types". 
This also reminded me I need Form info for Megas and G-Max, so added that as well in the most recent commit.